src/application-insights/azext_applicationinsights/_params.py

```diff
@@ -0,0 +1,44 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+from ._validators import validate_applications
+
+
+def load_arguments(self, _):
+    with self.argument_context('monitor app-insights metrics show') as c:
+        c.argument('application', options_list=['--app, -a'], id_part='name', help='GUID, app name, or fully-qualified Azure resource name of Application Insights component. The application GUID may be acquired from the API Access menu item on any Application Insights resource in the Azure portal. If using an application name, please specify resource group.')
+        c.argument('metric', options_list=['--metrics', '-m'], help='The metric to retrieve. May be either a standard AI metric or an application-specific custom metric.')
+        c.argument('timespan', options_list=['--timespan', '-t'], help='The timespan over which to retrieve metric values. This is an ISO8601 time period value. If timespan is omitted, a default time range of `PT12H` ("last 12 hours") is used. The actual timespan that is queried may be adjusted by the server based. In all cases, the actual time span used for the query is included in the response.')
+        c.argument('aggregation', help='The aggregation to use when computing the metric values. To retrieve more than one aggregation at a time, separate them with a comma. If no aggregation is specified, then the default aggregation for the metric is used.')
```

Comment:
The CLI only accepts Space-separated lists. If the service requires comma-separated anything, on the CLI-side you need to accept the space-separated list (nargs='+') and join them with commas.

Reply:
The functionality as implemented is correct, I'll update the help text.

Reply:
Fixed.